packages/jest/src/builders/jest/jest.impl.ts

```diff
@@ -69,7 +69,10 @@ function run(
     require.resolve('jest-preset-angular');
     Object.assign(tsJestConfig, {
```

Comment:
Right now it is not possible to modify these options from local workspace config. Would it make sense having `jest.config.js` and `jest-ng.config.js` in root dir of workspace?

`jest-ng.config.js`:
```js
module.exports = {
  preset: './jest.config.js',
  stringifyContentPathRegex: '\\.(html|svg)$',
  astTransformers: [
    'jest-preset-angular/build/InlineFilesTransformer',
    'jest-preset-angular/build/StripStylesTransformer'
  ],
  snapshotSerializers: [
    'jest-preset-angular/build/AngularSnapshotSerializer.js',
    'jest-preset-angular/build/HTMLCommentSerializer.js'
  ]
};
```

and then `libs/dummy/jest.config.js`:
```js
module.exports = {
  name: 'dummy',
  preset: '../../jest.config.js',
  coverageDirectory: '../../coverage/libs/dummy'
};
```

Reply:
I can see the sense of your suggestion, think that's more of a design decision would want maintainers to weigh in on; may also be more of its own PR vs part of this one as this is focused on just updating a dependency, like for like.